admin/broadleaf-open-admin-platform/src/main/resources/open_admin_style/css/admin/listGrid.css

```diff
@@ -33,6 +33,25 @@
     height: 45px;
     white-space: nowrap;
 }
+.listgrid-search .applied-filters-container {
```

Comment:
Let's make the added filters look akin to bootstrap nav pills (https://getbootstrap.com/docs/4.0/components/navs/#pills). Note that we don't have bootstrap 4 in open-admin, so you'll need to do it custom. You can probably borrow some of the styling from `selectize-admin-customization.css`:

```css
.selectize-control.multi .selectize-input [data-value] {
    background-color: #8EAA2B;
    border:none;
    margin: 5px 0 5px 5px;
}
```

Reply:
Also, we need to check what these added filters look like when using a listGrid for a collection and in a model. See Product custom attributes (bottom of first tab) and ProductOption lookup on Product's "Options" tab

Reply:
If we have something like `COLLECTION_IN` rule, then the pill could look like `Parent Category in: Home, Hot Sauces, Merchandise`